### ISSUE TITLE
Archive Data error

### ISSUE THREAD

**Issue by gotmilk0112** [ccd0/4chan-x#1853]
https://puu.sh/zGb7U/eefff86579.png

Getting this error every time I refresh or go to a different page.

**Comment by Liag** [comment]
Go to the advanced settings page and change the old archive list address to ``https://4chan-x.just-believe.in/json/archives.json``.

**Comment by ccd0** [comment]
You probably have another extension blocking the request. Temporarily disable everything else and turn them back on one by one so you can figure out what's doing it and possibly reconfigure it.

**Comment by gotmilk0112** [comment]
Haven't changed any of my extensions in a long time; not sure what would be blocking it.

The solution above worked, though.

**Comment by floppyD** [comment]
Got the same issue and haven't messed with extension in while too. It all started yesterday.

**Comment by MightyFo0l** [comment]
Same here, no new extensions or any changes

**Comment by tidux** [comment]
The entire https://mayhemydg.github.io/ site 404s.  Please replace with a different API endpoint upstream.

**Comment by ccd0** [comment]
The URL is
https://mayhemydg.github.io/archives.json/archives.json
and it loads fine for me.

**Comment by floppyD** [comment]
I just disabled and re-enabled uBlock Origin and got no errors so far. Weird bug,

**Comment by gotmilk0112** [comment]
Yeah, "https://mayhemydg.github.io/archives.json/archives.json" was giving errors again, then I just disabled uBlock, refreshed, then enabled uBlock and refreshed again, no error.

Weird shit.

**Comment by swablueme** [comment]
https://i.imgur.com/LflU3wr.png
Thanks to this issue I've realised that ublock has randomly decided to block the github and simply greening it out fixed it. It had exactly the same issue with a tumblr extension, xkit. Perhaps github has been added to the blacklist.

**Comment by ccd0** [comment]
Can you check the logger so you can see which filter list it's coming from?

**Comment by swablueme** [comment]
Weirdly enough ccd0 it's not coming up in the list of sites accessed at all, do you run the request on a schedule? https://i.imgur.com/ANkAANj.png

**Comment by ccd0** [comment]
Yeah, it checks every 2 days. Hit the "Update now" button in the Advanced settings to force an immediate check.

**Comment by swablueme** [comment]
@ccd0 
Here you are senpai https://i.imgur.com/RzKNFGn.png this is the filterlist

**Comment by ccd0** [comment]
It appears that filter already caused problems on other sites, and has been tweaked so it no longer causes problems.
https://forums.lanik.us/viewtopic.php?f=64&t=40185&p=133775

So if you update your filters, the issue should go away.

**Comment by Cormak** [comment]
Updated the filters on UBO and it worked. Easyprivacy filter was the culprit.

**Comment by lc-guy** [comment]
easylist FR still blocks it.

**Comment by neesema** [comment]
I have the same problem. I disabled every other addon besides violentmonkey and it I still have it. Could another script cause this conflict?

**Comment by ccd0** [comment]
#1915:
> Submitter: Dan
> Email: anguria_d@live.com
> Whenever I open a new thread or refresh i get the following message:
> 
> Error fetching archive data from
> https://mayhemydg.github.io/archives.json/archives.json
> Connection Error
> 
> This only happens on Chrome. With Firefox I don't get that error.

jonapoul:
> Pretty sure this is a problem with uBlock Origin. I had the same thing and it was being blocked by the [NoCoin filter list](https://github.com/hoshsadiq/adblock-nocoin-list) for disabling browser-based miners. 
> 
> The message goes away for me when I disable the filters, so take that as you will.
